src/test/java/com/microsoft/sqlserver/testframework/DBSchema.java

```diff
@@ -95,6 +95,19 @@
             Collections.shuffle(this.sqlTypes);
         }
     }
+    
+    /**
+     * 
+     * @param autoGenerateSchema
+     * @param alternateSchema
+     * @param dtSchema
+     */
+    DBSchema(boolean autoGenerateSchema, boolean alternateSchema, boolean dtSchema) {

```

Comment:
The same problem where not all the parameters are used. The existing DBSchema(boolean autoGenerateSchema) has covered `sqlTypes.add(new SqlDateTime2());`.